municipalities/beek-design-tokens/src/component/amsterdam/pagination.tokens.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
The pagination needs to look like the pagination here https://www.gemeentebeek.nl/zoeken?trefwoord=nieuws